sections/errorhandling/testingerrorflows.md

```diff
@@ -35,4 +52,28 @@ it("Creates new Facebook group", function (done) {
     done();
   });
 });
-```
+</pre>
+</details>
+
+<details>
+<summary><strong>Typescript</strong></summary>
+<pre lang="typescript">
+it("Creates new Facebook group", async function (done) {
+  let invalidGroupInfo = {};

```

Comment:
Same thing here, I think that by "modernizing" the javascript code, we could remove the need for a Typescript example. WDYT?

Reply:
Sounds great, we should maybe do this in a separate PR 👍

Reply:
@BrunoScheufler @kevynb In that case, we won't include sometimes TS examples?

Reply:
Yes, in those case I think we could only display a JavaScript example. I think in some simple case having a typescript code example does not add value.

Reply:
@kevynb It might look inconsistent and fool readers to believe that it wasn't done yet. Some may contribute it

Maybe include a title 'TypeScript - same as JS'?

Reply:
I'm not sure it's a good idea to make the doc more verbose just to prevent a contributor from making a contrib. Maybe we should state somewhere that Typescript examples need to be added only where they add a value?

Reply:
Consistency is an important part of readability - people absorb the format and tends to look for the same. Anyway, that's not critical for now, let's go with your thoughts